### DISCUSSION TITLE
Scalar unit conversion

### DISCUSSION THREAD

**Discussion by MuongKimhong** [Textualize/textual#4370] [Q&A]
Is their anyway to convert scalar unit from one unit to another? 

For example 
```python
box_height = self.query_one("Box").styles.height.value # 30h 

box_height_as_cell = ? # convert 30h to cell unit or
box_height_as_fraction = ? # convert 30h to fraction unit

```
or 
`fraction to cell`, `cell to fraction`, etc.

Documentation doesn't mention anything related to unit conversion

**Comment by TomJGooding** [comment]
Couldn't you just check the `Widget.size.height`?
